Helm: ability to specify image pull secrets (#1619)

z3k-io opened the pull request:
### Description
Added global value `global.seleniumGrid.imagePullSecret`
Added per component override `component.imagePullSecret`

Updated readme
Bumped chart version to 0.7.0

### Motivation and Context
Allows users to deploy images from a private repository rather than dockerhub.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [X] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

z3k-io committed:
Helm: ability to specify image pull secrets (#1619)

Added global value `global.seleniumGrid.imagePullSecret`
Added per component override `component.imagePullSecret`

Updated readme
Bumped chart version to 0.7.0

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/docker-selenium?pullRequest=1620) <br/>All committers have signed the CLA.

diemol commented:
Hi @zmconnelly, I had to bump the chart version to acknowledge the recent image release. Can you please update your fork and then we can merge?

z3k-io committed:
Helm: ability to specify image pull secrets (#1619)

Added global value `global.seleniumGrid.imagePullSecret`
Added per component override `component.imagePullSecret`

Updated readme
Bumped chart version to 0.7.0

z3k-io committed:
Merge branch '1619-helm-image-pull-policy' of https://github.com/zmconnelly/docker-selenium into 1619-helm-image-pull-policy

z3k-io commented:
@diemol done, thanks!

z3k-io committed:
Merge branch 'trunk' into 1619-helm-image-pull-policy

diemol reviewed (approved):
Thank you, @zmconnelly!
